tests/stress/compute_flow_with_threads.py

```diff
@@ -200,19 +178,19 @@
     DATA_asset = ocean.assets.resolve(DATA_did)
     ALGO_asset = ocean.assets.resolve(ALGO_did)
 
-    compute_service = get_first_service_by_type(DATA_asset, "compute")
-    algo_service = get_first_service_by_type(ALGO_asset, "access")
+    compute_service = DATA_asset.services[0]
```

Comment:
I'm not sure about the business logic here. While it is true that currently these indices are correct, we specify in the readmes that this is just a coincidence, and the first compute/access index might be different for different DDOs. We made the decision a few iterations ago that we should keep `get_first_service_by_type` in tests. Should we revisit that decision?

Reply:
`get_first_service_by_type` is okay in tests where we know the asset only has 1 service of the given type.

If the asset has 2 services of the same type, then we should look up the asset using the index or the id.

```python
asset.get_service_by_index()
asset.get_service_by_id()
```